src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
-export interface Figure {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+function checkLength(...args: number[]): void {
+  if (args.some((side) => side <= 0)) {
+    throw new Error('Please, enter valid values');
```

Comment:
It's not an error, it's a user prompt.
```suggestion
    throw new Error('Incorrect figure parameters: all values need to be > 0!');
```